tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,15 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel
+
+require('babel-core/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
Oh yeah, the other point is that `register` is actually calling `install` for a reason: it itself overrides `retrieveSourceMap` to load maps from an in-memory hash that it fills when it compiles things.  Do source maps on the tool files even work when you override that retrieveSourceMap?

I'm guessing the answer is yes because `register` passes `sourceMap: "both"` which I'm guessing means "both output the map externally *and* inline", and you end up using the inline map.  And we just waste some ram storing a second copy of the source map in memory (when running from a checkout).  So not the end of the world but not great either?